signac_dashboard/views.py

```diff
@@ -89,13 +103,34 @@ def get_file(dashboard, jobid, filename):
             abort(404, "The file requested does not exist.")
 
 
+def get_project_file(dashboard, filename):
+    if dashboard.project.isfile(filename):
+        mimetype = None
+        cache_timeout = 0
+        # Return logs as plaintext
+        textfile_regexes = ["job-.*\\.[oe][0-9]*", ".*\\.log", ".*\\.dat"]
+        for regex in textfile_regexes:
+            if re.match(regex, filename) is not None:
+                mimetype = "text/plain"
+        return send_file(
```

Comment:
1. Does this work for paths that are symlinks?

2. Can this URL endpoint be used to read a signac config? (Is that a potential security risk, e.g. if the config contains signac-flow account information?)